releases/2023.2/2023.2.md

```diff
@@ -0,0 +1,182 @@
+[kytos-utils](https://github.com/kytos-ng/kytos-utils)              | [kytos-utils](https://github.com/kytos-ng/kytos-utils/releases)
+[ui](https://github.com/kytos-ng/ui)                                | [ui](https://github.com/kytos-ng/ui/releases)
+[kytos/of_core](https://github.com/kytos-ng/of_core)                | [kytos/of_core](https://github.com/kytos-ng/of_core/releases)
+[kytos/flow_manager](https://github.com/kytos-ng/flow_manager)      | [kytos/flow_manager](https://github.com/kytos-ng/flow_manager/releases)
+[kytos/topology](https://github.com/kytos-ng/topology)              | [kytos/topology](https://github.com/kytos-ng/topology/releases)
+[kytos/of_lldp](https://github.com/kytos-ng/of_lldp)                | [kytos/of_lldp](https://github.com/kytos-ng/of_lldp/releases)
+[kytos/pathfinder](https://github.com/kytos-ng/pathfinder)          | [kytos/pathfinder](https://github.com/kytos-ng/pathfinder/releases)
+[amlight/coloring](https://github.com/kytos-ng/coloring)            | [kytos-ng/coloring](https://github.com/kytos-ng/coloring/releases)
+[amlight/sdntrace_cp](https://github.com/kytos-ng/sdntrace_cp)      | [kytos-ng/sdntrace_cp](https://github.com/kytos-ng/sdntrace_cp/releases)
+[kytos/mef_eline](https://github.com/kytos-ng/mef_eline)            | [kytos/mef_eline](https://github.com/kytos-ng/mef_eline/releases)
+[kytos/of_multi_table](https://github.com/kytos-ng/of_multi_table)  | [kytos/of_multi_table](https://github.com/kytos-ng/of_multi_table/releases)
+[kytos/telemetry_int](https://github.com/kytos-ng/telemetry_int)    | [kytos/telemetry_int](https://github.com/kytos-ng/telemetry_int/releases)
+
+### Switches Compatibility List
+
+The following switches have been tested with Kytos version `2023.2`:
+
+Switch Vendor                        | Switch          | Firmware version        | OpenFlow version | Environment type
+-------------------------------------|-----------------|-------------------------|------------------|-----------------
+[OVS](https://www.openvswitch.org/)  | ovs-switchd     | 2.15                    | 1.3              | testing
+[NoviFlow](https://www.noviflow.com) | WB5132-F        | NW570.2.1 and NW550.2.3 | 1.3              | production
+
+#### Kytos Testing Dashboard Analytics
+
+An overview of the Kytos-ng and end-to-end tests coverage is available on these dashboards:
+
+- [Kytos-ng Tests Analytics Scrutinizer](https://kytos-tests.amlight.net/index.html)
+- [AmLight Tests Analytics GitLab CI](https://kytos-tests.amlight.net/amlight.html)
+
+Unit test code coverage has increased to at least 90% on most NApps.
+
+## GitHub statistics
+
+In the [kytos-ng](https://github.com/kytos-ng) organization, during the period of this release from August 2023 to February 2024:
+
+- [161 pull requests were merged](https://github.com/search?q=org%3Akytos-ng+is%3Apr+is%3Aclosed+merged%3A2023-08-01..2024-03-01&type=Issues)
```

Comment:
I'll need to update this stats, but since it might change if a PR needs to land, I'll do it right before merging this PR in the next weeks or so.